src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_XmlDoc.vb

```diff
@@ -625,6 +625,54 @@ class c
             End Using
         End Function
 
+        <WpfTheory, CombinatorialData>
+        Public Async Function InvokeWithOpenAngleSeeCommitLangwordWithEqualsQuotes(showCompletionInArgumentLists As Boolean) As Task
+
+            Using state = TestStateFactory.CreateCSharpTestState(
+                <Document><![CDATA[
+class c
+{
+    /// <summary>
+    /// <see $$=""
+    /// </summary>
+    void goo() { }
+}
+            ]]></Document>, showCompletionInArgumentLists:=showCompletionInArgumentLists)
+
+                state.SendTypeChars("l")
+                state.SendInvokeCompletionList()
+                Await state.AssertSelectedCompletionItem(displayText:="langword")
+                state.SendReturn()
+
+                ' /// <see langword="$$"/>
```

Comment:
```suggestion
                ' /// <see langword="$$"
```
I appreciate the effort to show the result in a more readable form, but it should precisely match the actual outcome